caktus_primer/static/js/modal.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
```

Comment:
ES6 doesn't need this directive, because they are strict by default.

Reply:
🙌